### PR TITLE
Remove $s from README.md

### PR DESCRIPTION
**Pull request by Radolumbo**
## Summary

The presence of `$` in all the env var names is confusing -- when you set an environment variable, you don't put `$BPE_MY_ENV=value` .. you just put `BPE_MY_ENV=value`. $ implies the dereferencing of the env var, which opens up the documentation to misinterpretation here. However, the `$NAME` parts of the docs are correctly indicating that it will actually dereference the value of NAME and set appropriately.

## Use Cases

Easier understanding of the docs.

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by dmikusa-pivotal** [comment]
I understand the technical argument being made here. I think the reason we prefix things with `$` is to further emphasize that it's an env variable. It's debatable if that is having the intended effect or not. I do think it can make copy & paste harder in some cases.

I'm not opposed to changing it, but we can't change this only in one buildpack. It would have to be done across all the buildpacks & in the Paketo docs so that things are consistent. Are you interested in submitting PRs for more buildpacks? If not, we can take an action item to make this change but it would be a low priority change.

Thanks

**Review by robdimsdale — APPROVED** [review]

**Comment by robdimsdale** [comment]
Whoops, I accidentally approved this. I can't dismiss my own review because I'm not on the utilities team.

@paketo-buildpacks/utilities-maintainers can you dismiss my review please?

Also, given this PR has had no response for about 9 months, perhaps you could consider closing it?

**Comment by dmikusa** [comment]
>I'm not opposed to changing it, but we can't change this only in one buildpack. It would have to be done across all the buildpacks & in the Paketo docs so that things are consistent.

Closing for this reason. We can't change this in one place only. We need it to be consistent. If there's interest in making this change, let's come up with a plan to PR this across all of the buildpacks & Paketo Docs.

**Pull request closed without merging**
